--- /etc/bind/named.conf.options.orig	2014-06-17 10:57:17.000000000 +0200
+++ /etc/bind/named.conf.options	2014-06-22 00:29:00.204787007 +0200
@@ -20,6 +20,8 @@
 	//========================================================================
 	dnssec-validation auto;
 
+  zone-statistics yes;
+  statistics-file "/var/lib/bind/named.stats";
   version "none";
 	auth-nxdomain no;    # conform to RFC1035
 	listen-on-v6 { any; };
